server/startup/serverRunning.js

```diff
@@ -16,13 +16,16 @@ Meteor.startup(function() {
 		}
 	}
 
+	const mongoDbVersion = MongoInternals.NpmModules.mongodb.version;
```

Comment:
are you sure this gets the version of the mongo server? we used [this code](https://github.com/RocketChat/Rocket.Chat/blob/develop/app/statistics/server/functions/get.js#L130) to get it for statistics.. maybe this should use the same.

Reply:
no. 😂 because it didn't.  Updated thanks!